code/datums/supplypacks.dm

```diff
@@ -1657,6 +1657,18 @@ AMMO
 	containername = "\improper smartgun powerpack crate"
 	group = "Ammo"
 
+/datum/supply_packs/ammo_smartmachinegun
+	name = "T26 smartmachinegun ammo crate (x3)"
+	contains = list(
+                    /obj/item/ammo_magazine/standard_smartmachinegun,
+                    /obj/item/ammo_magazine/standard_smartmachinegun,
+					/obj/item/ammo_magazine/standard_smartmachinegun
+					)
+	cost = 30
```

Comment:
Are you using tabs or spaces here?

Reply:
should've fixed this I think. I did an oops.